File: tamper-scripts/ruby/base64encode.rb
#!/usr/bin/env ruby
#
# Author:       KING SABRI | @KINGSABRI
# Description:  Base64 encoding all characters in a given payload
# Requirements: None
#
require 'json'
require 'base64'

@json    = JSON.parse(ARGV[0])
@payload = @json["payload"]
@kwargs  = @json["kwargs"]

@json["payload"] = Base64.urlsafe_encode64(@payload)

print @json.to_json

File: tamper-scripts/javascript/base64encode.js
#!/usr/bin/env nodejs
//
// Author:       KING SABRI | @KINGSABRI
// Description:  Base64 encoding all characters in a given payload
// Requirements: None
//
var json    = JSON.parse(process.argv[2]);
var payload = json["payload"];
var kwargs  = json["kwargs"];

process.stdout.write(JSON.stringify(json));
